client/layout/logged-out.jsx

```diff
@@ -133,7 +139,10 @@ export default withCurrentRoute(
 		const sectionName = currentSection?.name ?? null;
 		const sectionTitle = currentSection?.title ?? '';
 		const isJetpackLogin = currentRoute.startsWith( '/log-in/jetpack' );
-		const isWhiteLogin = currentRoute.startsWith( '/log-in/new' );
+		const isPartnerSignup = isPartnerSignupQuery( currentQuery );
+		const isPartnerSignupStart = currentRoute.startsWith( '/start/wpcc' );
+		const isWhiteLogin =
+			currentRoute.startsWith( '/log-in/new' ) || ( isPartnerSignup && ! isPartnerSignupStart );
```

Comment:
Not enjoying this hack on top of a hack setup but it's required if we want to fix the flash of woo text when transitioning to the user creation screen before completing the rebrand of that page (https://github.com/Automattic/wp-calypso/pull/64477)